client/components/BookingContainer.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import $ from 'jquery';
+import moment from 'moment';
+import { CalendarRow } from './CalendarRow.jsx';
+import { Booking } from './Booking.jsx';
+
+class BookingContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dates: [],
+      selectMonthCalendar: [],
+      selectDates: [],
+      availabilityModel: [],
+    }
+    this.getDateInfoForSelectedYear = this.getDateInfoForSelectedYear.bind(this);
+    this.determineStructureOfCalendarForSelectedMonth = this.determineStructureOfCalendarForSelectedMonth.bind(this);
+    this.buildCalendarRowForSelectedMonth = this.buildCalendarRowForSelectedMonth.bind(this);
+    this.getDummyAvailabilityFigureForRoom = this.getDummyAvailabilityFigureForRoom.bind(this);
+    this.modelAvailabilityForSelectedRoom = this.modelAvailabilityForSelectedRoom.bind(this);
+    this.onCalendarChangeClick = this.onCalendarChangeClick.bind(this);
+  }
+  componentDidMount() {
+    this.getDateInfoForSelectedYear(moment().year());
+    this.getDummyAvailabilityFigureForRoom();
+  }
+  getDummyAvailabilityFigureForRoom() {
+    /* 
+    This will provide the room id from the window that we need for the initial GET
+    Since were not live yet the dummy below will serve the same purpose. 
+
+    const parsedWindowURL = window.location.pathname.split('/');
+    const userRoomId = parsedWindow[parsedWindow.length - 1];
+    */
+    const dummyUserRoomPath = '/rooms/70';
+
+    $.ajax({
+      url: `${dummyUserRoomPath}`,
+      method: 'GET'
+    })
+      .done((res) => {
+        res = JSON.parse(res);
+        console.log(res.availabilityScore);
+        this.modelAvailabilityForSelectedRoom(res.availabilityScore);
+      });
+  }
```

Comment:
We want the static file to be served only for the path ‘/rooms/:id/‘. So you can state that the static file will be served only for that path (app.use('/rooms/:id/', express.static(path.join(__dirname, <path to your index.html file>)))

Reply:
You can get room number from window.location.pathname and pass it as props to the Booking component

Reply:
Hi @Lamanova requested changes have been made. Please check & provide approval.